app/helpers/application_helper.rb

```diff
@@ -36,4 +43,19 @@ def decorator_instance(object, decorator_class = nil)
     decorator_class ||= "#{object.class.to_s.demodulize}Decorator".constantize
     decorator_class.new(object, self)
   end
+
+  def column_sort_icon
+    sort_direction == 'asc' ? "\u25B2" : "\u25BC"
+  end
+
+  def column_title(column)
```

Comment:
ideally part of a decorator

Reply:
👍 I have moved column_title to the prosecution_case_decorator